frontend/views/pages/Contributions.vue

```diff
@@ -2,167 +2,164 @@
 page(pageTestName='contributionsPage' pageTestHeaderName='contributionsTitle')
   template(#title='') {{ L('Contributions') }}
 
-  // v-if='groupProfile(ourUsername).incomeDetailsKey'
   callout-card(
     :isCard='true'
+    data-test='addIncomeDetailsFirstCard'
     :title='L("Add your income details")'
     :svg='SvgContributions'
+    v-if='!memberGroupProfile.incomeDetailsType'
   )
     i18n(tag='p') This will allow you to start receiving or giving mincome.
-    i18n(tag='button' @click='openModal("IncomeDetails")') Add income details
+    i18n(
+      tag='button'
+      data-test='openIncomeDetailModal'
+      @click='openModal("IncomeDetails")'
+    ) Add income details
 
-  section.card
-    i18n(tag='h2' class='card-header') Receiving
+  template(v-else)
+    .c-contribution-header
+      .has-text-1
+        i18n(
+          v-if='needsIncome'
+          tag='p'
+          data-test='headerNeed'
+          :args='{upTo: upTo,bold_:\'<span class="has-text-bold has-text-0">\',_bold:\'</span>\'}'
+        ) You need {bold_}{upTo}{_bold}
```

Comment:
The pattern: `{tag_}{variable}{_tag}` should be rewritten using just `{variable}` to make life less confusing for the translators. Tags are useful when the thing between them is a word to be translated, i.e. `{tag_}english word or fragment that is not a variable{_tag}`.

So here you can do:

```
        i18n(
          v-if='needsIncome'
          tag='p'
          data-test='headerNeed'
          :args='{ amount: `<span class="has-text-bold has-text-0">${upTo}</span>` }'
        ) You need {amount}
```

The same goes for the two `i18n`s below and any other ones in this PR that follow the pattern `{tag_}{variable}{_tag}`